docs/sphinx/source/api.rst

```diff
@@ -102,8 +102,8 @@ Airmass and atmospheric models
    :toctree: generated/
 
    location.Location.get_airmass
-   atmosphere.absoluteairmass
-   atmosphere.relativeairmass
+   atmosphere.absolute_airmass
```

Comment:
pvlib only includes one absolute airmass model so it didn't occur to me to add ``get_`` here. I'm fine with either approach at this point.

Reply:
I see your point.  It may help when someone is trying to find the relevant function, to keep names parallel.  There are other models for absolute air mass which we haven't implement.